### ISSUE TITLE
What is the meaning of the number in the name of the result folders?

### ISSUE THREAD

**Issue by SorryGeats** [TQTQliu/MVSGaussian#21]
Hello sir, would you mind telling me the the meaning of the number in the name of the result folders?
![image](https://github.com/user-attachments/assets/8bee6cf7-029d-423d-b69d-bc971224f447)

**Comment by SorryGeats** [comment]
And it is apprieciated for you to explain the detail about what does the metric like PSNR、SSIM calculate specificly in your work? Does it derived form the input images and the novel view images?

**Comment by TQTQliu** [comment]
Hello, sorry for the late reply.
Let's first review the setup for the evaluation. Given N views of a scene, we first divide them into working sets and test sets. During testing, our goal is to synthesize the target view at the test viewpoint. We will select 2 or 3 nearby views from the working set as the input source view and then output the target view. Metrics such as PSNR and SSIM are calculated by comparing the synthesized views with the Ground-Truth views from the test set.
​
When running the command to synthesize the video, `{scene_name}_{test_view_id}_0.mp4` will be generated, which represents the video centered at the `test_view_id` viewpoint. Accordingly, the folder `{scene_name}_{test_view_id}` contains all the generated images, that is, all the frames of the above video.

**Comment by SorryGeats** [comment]
Thank you for the detailed explanation of the evaluation setup. I appreciate the clarity in outlining the process of dividing the views into working sets and test sets, and the steps involved in synthesizing the target view. The information about the metrics calculation and the generation of the video and image frames is very helpful. Thanks again for your thorough explanation!

**Comment by SorryGeats** [comment]
Hello sir, can you explain the `train_dataset` and `test_datase` in the `colmap_eval.yaml`?
![image](https://github.com/user-attachments/assets/841e6976-8e01-4d30-bd61-a24ccd234fbb)

**Comment by SorryGeats** [comment]
And I wonder why it does not display the traning process and the ssim and psnr?
![image](https://github.com/user-attachments/assets/013484ed-f971-4595-9f47-87eada6ac36f)

**Comment by TQTQliu** [comment]
`train_dataset` and `test_dataset` represent the setup for training and test, respectively, such as data path, image resolution, and so on.
The reason the training is not executed is because the loaded checkpoint `./trained_model/mvsgs/dtu_pretrain/latest.pth` has reached the predefined number of [epochs](https://github.com/TQTQliu/MVSGaussian/blob/aa5e786c865858f42c7942edd931cc3b79825628/configs/mvsgs/dtu_pretrain.yaml#L66).

**Comment by SorryGeats** [comment]
> `train_dataset` and `test_dataset` represent the setup for training and test, respectively, such as data path, image resolution, and so on. The reason the training is not executed is because the loaded checkpoint `./trained_model/mvsgs/dtu_pretrain/latest.pth` has reached the predefined number of [epochs](https://github.com/TQTQliu/MVSGaussian/blob/aa5e786c865858f42c7942edd931cc3b79825628/configs/mvsgs/dtu_pretrain.yaml#L66).

So how can I realize for each new scene, I can see its training iterations in the log files, and save its evaluation indicators？
Do you mean I have to delete the trained_model .pth files to realize my goals?
![image](https://github.com/user-attachments/assets/57a455b6-94d7-473f-bdaf-ef724be1f4d8)

**Comment by SorryGeats** [comment]
> You can modify the [`exp_name`](https://github.com/TQTQliu/MVSGaussian/blob/823713141181fd68ef05ab188ed36bf7f1045ea5/configs/mvsgs/dtu_pretrain.yaml#L3) in the `configs/mvsgs/dtu_pretrain.yaml`.
> 
> Before training, the code will first check whether there is checkpoint in `trained_model/mvsgs/exp_name`, and if so, the latest checkpoint will be loaded. During training, the tensorboard log will be save in `record/mvsgs/exp_name`, the trained checkpoint will be save in `trained_model/mvsgs/exp_name`, and the rendering results will be saved in `result/mvsgs/exp_name`.

Ah you're right! That is a proper way to avoid the dilemma.
